devnet-sdk/systemgo/system_test.go

```diff
@@ -0,0 +1,58 @@
+package systemgo
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ethereum/go-ethereum/log"
+
+	"github.com/ethereum-optimism/optimism/devnet-sdk/system2"
+	"github.com/ethereum-optimism/optimism/op-service/testlog"
+)
+
+func TestSystem(t *testing.T) {
+	ids, opt := DefaultInteropSystem(ContractPaths{
+		FoundryArtifacts: "../../packages/contracts-bedrock/forge-artifacts",
+		SourceMap:        "../../packages/contracts-bedrock",
+	})
```

Comment:
Is this also just part of the work in progress, because we obviously don't want every test to have to specify this. Also not really surprising if this system test is lower level than what actual acceptance tests would actually have to do. Just trying to improve my understanding of how everything winds up fitting together.

Reply:
Yes, work-in-progress, it's captured here: https://github.com/ethereum-optimism/optimism/issues/15026